test/link/load-fragment/section-fragment/ref.out.map

```diff
@@ -0,0 +1,24 @@
+SUMMARY:
+	ROM0: 20 bytes used / 16364 free
+	WRAM0: 8 bytes used / 4088 free
+
+ROM0 bank #0:
+	SECTION: $0000-$0013 ($0014 bytes) ["rom"]
+	         $0000 = Part1
+	         $0004 = Part1End
+	         ; Next fragment
+	         $0010 = Part3
+	         $0014 = Part3End
+	         ; Next fragment
+	         $0004 = Part2
+	         $0010 = Part2End
```

Comment:
These fragments are not output in the correct order. (Check if this affects all of `SECTION FRAGMENT` too.)